### DISCUSSION TITLE
OnlyOffice

### DISCUSSION THREAD

**Discussion by jasonpearce** [runtipi/runtipi-appstore#2399] [Requests]
### Description

OnlyOffice is a free software office suite and ecosystem of collaborative applications. It features online editors for text documents, spreadsheets, presentations, forms and PDFs, and the room-based collaborative platform.

### Official Website

https://www.onlyoffice.com/

### Documentation link

https://www.onlyoffice.com/developer-edition.aspx

### Last application release & date

8.0.0 on Feb 1, 2024

### Application license

AGPL-3.0 license

### Source code repository link

https://github.com/ONLYOFFICE/DesktopEditors

### Docker image link

https://github.com/ONLYOFFICE/Docker-DocumentServer

### Other's

Although OnlyOffice is already included in the Kasm Workspaces app, it would be more accessible as a standalone app.

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Duplicate of https://github.com/runtipi/runtipi-appstore/discussions/2215

**Reply by jasonpearce** [reply]
You're right. So sorry.
